tests/vmi_configuration_test.go

```diff
@@ -310,7 +310,7 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 					}
 				}
 				Expect(computeContainer).ToNot(BeNil(), "could not find the compute container")
-				Expect(computeContainer.Resources.Requests.Memory().ToDec().ScaledValue(resource.Mega)).To(Equal(int64(416)))
+				Expect(computeContainer.Resources.Requests.Memory().ToDec().ScaledValue(resource.Mega)).To(Equal(int64(399)))
```

Comment:
IMO that's a weird metric to assert on. How does it relate to `should report 3 cpu cores under guest OS`?

Reply:
Each core adds a bit of memory overhead IIRC... We just request 16MiB less memory in no-graphics.

Reply:
Magic number alert.

Reply:
Indeed